sites/docs/docs/data-file-types.mdx

```diff
@@ -561,6 +561,205 @@ coordinationValues={`{
 }`}
 />
 
+
+## `obsFeatureMatrix.mudata.zarr`
```

Comment:
I think we should explain (if I'm understanding this correctly) that we use the shared index/var for loading this matrix.  That's correct?  And that you can use this as a "normal" `AnnData` object otherwise.

Reply:
Same goes for other things.

Reply:
In this implementation we use the modality-specific indices, but that is a good point, we should document that behavior.